components/feedstock/providers.tsx

```diff
@@ -21,8 +35,11 @@ export const Providers: React.FC<ProvidersProps> = ({ providers }) => {
             href={provider.url}
```

Comment:
Not sure why, but it looks like these links are broken [here](https://leap-data-catalog-git-feedstock-page-carbonplan.vercel.app/feedstock/ldeopco2-hpd-back-in-time).